tests/test_snapcraft_store_api.py

```diff
@@ -0,0 +1,138 @@
+from vcr_unittest import VCRTestCase
+from canonicalwebteam.store_api.stores.snapcraft import SnapcraftStoreApi
+
+
+class SnapcraftApiTest(VCRTestCase):
+    def setUp(self):
+        # configure self.attribute
+        self.client = SnapcraftStoreApi()
+        return super(SnapcraftApiTest, self).setUp()
+
+    def test_get_all_items(self):
+        result = self.client.get_all_items(size=16, api_version=1)
+        snaps = result["_embedded"]["clickindex:package"]
```

Comment:
This is pretty yuck. If this is the way the response from the API is formatted, I think it would be nicer to hide this ugliness inside the function and return to the client a nice list of snaps. But don't change this now, I think it's out-of-scope for this PR.

Reply:
I totally agree. I think this module should have moved that part of the code from Snapcraft:

https://github.com/canonical-web-and-design/snapcraft.io/blob/master/webapp/store/logic.py#L18-L29

Reply:
Filed https://github.com/canonical-web-and-design/canonicalwebteam.store-api/issues/11